app/lib/three_scale/redis_config.rb

```diff
@@ -6,15 +6,10 @@ def initialize(redis_config = {})
       raw_config = (redis_config || {}).symbolize_keys
       sentinels = raw_config.delete(:sentinels).presence
       raw_config.delete_if { |key, value| value.blank? }
+      set_pool_size(raw_config)
 
       @config = ActiveSupport::OrderedOptions.new.merge(raw_config)
       config.sentinels = parse_sentinels(sentinels) if sentinels
-
-      # The ID is forced to be nil to disable the default behavior in Sidekiq < 6
-      # which invokes CLIENT SETNAME command, which incompatible with some Redis providers
-      # see https://issues.redhat.com/browse/THREESCALE-9210
-      # This can be removed when Sidekiq is upgraded to a version >= 6.0.7
-      config.id = nil
```

Comment:
I didn't check this, but if the comment is true, we can safely remove this. Nothing seem to have failed after I removed it.

Reply:
Let's trust our forefathers hehe

Reply:
That was AWS redis I think. Not sure how hard to try it out.

Reply:
Yes, this is good :) 
https://github.com/3scale/porta/pull/3197 for context with more links to Sidekiq repo that explains the change.